### PR TITLE
toilet: add HEAD

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/toilet.rb [hunk 1]
```diff
@@ -15,10 +16,13 @@ class Toilet < Formula
     sha256 "ef2c34f742b366f84d2aeeb6d83fb94d6bd443f210e56968fb8b2b5700eab759" => :mavericks
   end
 
+  depends_on "autoconf" => :build if build.head?
+  depends_on "automake" => :build if build.head?
```
Comment (issyl0) [inline Formula/toilet.rb]:
```suggestion
  if build.head?
    depends_on "autoconf" => :build
    depends_on "automake" => :build
  end
```

**Review by issyl0 — COMMENTED** [review]
Apart from one tiny style nit this looks good to me!

File: Formula/toilet.rb (hunk 1 above)
Comment (Bo98) [reply Formula/toilet.rb]: Or better yet, put it in a `head do`:

```suggestion
  head do
    url "https://github.com/cacalabs/toilet.git"
    
    depends_on "autoconf" => :build
    depends_on "automake" => :build
  end
  
```

and remove the `head` line further up.

Comment (chenrui333) [reply Formula/toilet.rb]: Nice!

Comment (pthariensflame) [reply Formula/toilet.rb]: Implemented the latter!  Thanks @Bo98 and @issyl0!

**Comment by issyl0** [comment]
Thank you, @pthariensflame!

**Pull request merged**